haystack/components/converters/openapi_functions.py

```diff
@@ -3,9 +3,7 @@
 from pathlib import Path
 from typing import Any, Dict, List, Optional, Union
 
-import requests
 import yaml
-from requests import RequestException
 
 from haystack import Document, component, logging
```

Comment:
I think we can remove this now?
```suggestion
from haystack import component, logging
```